DCF_Demo.md

```diff
@@ -0,0 +1,554 @@
+# DCF Invoke Device Method Using Azure CLI
+
+## Open up an instance of Azure CLI 
```

Comment:
There should be a section here describing the steps one should follow before they can start this demo. (i.e. following the stm getting started guide)